axlearn/common/trainer.py

```diff
@@ -1059,19 +1059,25 @@ def _maybe_stop_or_start_tracing(
 
 
 def select_mesh_config(trainer_config: SpmdTrainer.Config, *, mesh_selector: str):
-    """Selects a mesh rule (if one matches `mesh_selector` to override mesh config.
+    """Selects a mesh rule (if one matches mesh_selector to override mesh config.
 
-    If any of `trainer_config.mesh_rules` matches `mesh_selector`, modifies
-    `trainer_config.mesh_shape` according to the rule.
+    If any of trainer_config.mesh_rules matches mesh_selector, modifies
```

Comment:
Can we retain the backticks for code formatting?

Reply:
John suggested me to take away the quotes, I can revert the comments here.